webhook_features/src/features/summary_comment.rs

```diff
@@ -286,23 +283,20 @@ impl AckType {
     }
 }
 
-macro_rules! multi_vec {
-    ($([$($key:literal),+] => $value:expr);*) => {
-        vec![
-            $($(($key, $value)),*),*
-        ]
-    };
-}
-
 lazy_static! {
-    static ref ACK_PATTERNS: Vec<(&'static str, AckType)> = multi_vec![
-        ["code review ack", "cr ack", "cr-ack", "crack"] => AckType::Ack;
-        ["ack", "utack", "tack"] => AckType::Ack;
-        ["concept ack", "concept-ack", "conceptack", "ack"] => AckType::ConceptAck;
-        ["concept nack", "concept-nack", "conceptnack"] => AckType::ConceptNack;
-        ["approach ack", "approach-ack", "approachack"] => AckType::ApproachAck;
-        ["approach nack", "approach-nack", "approachnack"] => AckType::ApproachNack;
-        ["nack"] => AckType::ConceptNack
+    static ref ACK_PATTERNS: Vec<(&'static str, AckType)> = vec![
```

Comment:
Not sure if possible, but can the regex be compiled static?

Reply:
Yes, better.